hikey.mk

```diff
@@ -351,7 +379,7 @@ nvme-cleaner:
 .PHONY: flash
 flash:
 	python $(ROOT)/burn-boot/hisi-idt.py --img1=$(LLOADER_PATH)/l-loader.bin
```

Comment:
You need `sudo` here or else one needs to do `sudo make flash` (in which case some more code than necessary is run as root...).